src/tutorial.md

```diff
@@ -651,6 +651,10 @@ Now we're ready to use our dapp!
 
 The easiest way to interact with our dapp in a browser is through [MetaMask](https://metamask.io/), a browser extension for both Chrome and Firefox.
 
+<p class="alert alert-info">
+<strong>Note</strong>: If you already use Metamask, you'll need to switch accounts. To do this, you can click your account logo and then "Lock". After this you can "import using Secret Recovery Phrase" to insert the Ganache mnemonic.
+</p>
```

Comment:
This is a good idea @zeosamaster !  Can you add a prompt to remind them to save their original mnemonic in order to switch back? Perhaps a link to [MetaMask's Basic Safety and Security Tips](https://metamask.zendesk.com/hc/en-us/articles/360015489591-Basic-Safety-and-Security-Tips-for-MetaMask) as well.

Reply:
Nice sugestion @cds-amal ! What do you think of this?
```suggestion
<strong>Note</strong>: If you already use Metamask, you'll need to switch accounts. Before doing that, make sure that you have your personal secret phrase backed up, since you'll need it to log back into your personal account (This secret phrase should definitely remain secret and you won't be able to access your account without it! Check [MetaMask's Basic Safety and Security Tips](https://metamask.zendesk.com/hc/en-us/articles/360015489591-Basic-Safety-and-Security-Tips-for-MetaMask) for more info.) After having your secret phrase safely backed up, you can safely switch to Ganache's test account. To do this, you can click your account logo and then "Lock". After this you can "import using Secret Recovery Phrase" to insert the Ganache mnemonic.
</p>
```

Reply:
I like it! Thank you!

Reply:
For some reason I couldn't commit your suggestion. Can you update please?

Reply:
Done, you can review :)